README.md

```diff
@@ -31,12 +31,42 @@ AVD Documentation:
 
 [Arista Networks](https://www.arista.com/) supports Ansible for managing devices running Arista's **Extensible Operating System (EOS)** natively through it's **EOS API (eAPI)** or [**CloudVision Portal (CVP)**](https://www.arista.com/en/products/eos/eos-cloudvision). The collection includes a set of Ansible roles and modules to help kick-start your automation with Arista. The various roles and templates provided are designed to be customized and extended to your needs.
 
+Full documentation for the collection:
```

Comment:
Do we need to add all this on the main page readme? It is redundant to the new index.md.

Reply:
Carl and I talked a bit about this. We need to revamp this readme to get the main points for anyone visiting on GitHub vs the main site. I agree it is very redundant @carlbuchmann

Reply:
If you check the history of this file, you removed all this stuff a while back. IMO the readme on github should be very basic - even more than what we have Today, and then link to the avd site. Maybe once we move the docs out of the collection, we can move the new `index.md` to be `README.md` for the repo.

Reply:
Agreed that is should be basic

Reply:
Took an initial try at modifying the root readme. Happy for any additional comments or what else we can trim.

Reply:
Agreed with @ClausHolbechArista keep this basic and should point back to avd.arista.com and also TAC support link.